src/Commands/RollerOff.cpp

```diff
@@ -0,0 +1,53 @@
+// RobotBuilder Version: 2.0
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// C++ from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+
+#include "RollerOff.h"
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
+
+RollerOff::RollerOff(): Command() {
+        // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+	Requires(Robot::shooter.get());
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+}
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
+
+// Called just before this Command runs the first time
+void RollerOff::Initialize() {
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void RollerOff::Execute() {
+	RobotMap::shooterrollerMotor->Set(0.0);
+
+	RobotMap::shootershooterMotor1->Set(Robot::oi->getdriver()->GetRawAxis(3));
+	RobotMap::shootershooterMotor2->Set(Robot::oi->getdriver()->GetRawAxis(3));
```

Comment:
Should we keep? When no commands are being called on the shooter subsystem command this will run, taking in the value of the trigger and setting it to the motors.